xlml/apis/test_config.py

```diff
@@ -136,11 +136,13 @@ class TpuVmTest(TestConfig[Tpu]):
     test_name: Unique name for this test/model.
     set_up_cmds: List of commands to run once when TPU is created.
     run_model_cmds: List of commands to run the model under test.
+    num_slices: Number of TPU slices.
   """
 
   test_name: str
   set_up_cmds: Iterable[str]
   run_model_cmds: Iterable[str]
+  num_slices: int = 1
 
   @property
   def benchmark_id(self) -> str:
```

Comment:
Could you change this to `return f'{self.test_name}-{self.num_slices}x{self.accelerator.name}'` so that the benchmark_id contains the number of slices? Same with in `TpuGkeTest`

Reply:
I agree with Jon! For `return f'{self.test_name}-{self.num_slices}x{self.accelerator.name}'`, it will be great to just apply to multislice tests.

Reply:
Done.